packages/vuetify/src/presets/default.ts

```diff
@@ -34,7 +34,7 @@ const preset: VuetifyPreset = {
     },
   },
   icons: {
-    iconfont: 'md'
+    iconfont: 'svg'
```

Comment:
Shouldn't it be `mdi`, the same as on packages/vuetify/src/services/icons/index.ts line 13?

Reply:
fixed